frontend/src/components/SideNavigationTeamOptions.vue

```diff
@@ -95,7 +95,7 @@ export default {
                     to: '/library',
                     tag: 'shared-library',
                     icon: FolderIcon,
-                    featureUnavailable: !this.features?.['shared-library']
+                    featureUnavailable: !this.features?.['shared-library'] || this.team?.type.properties.features?.['shared-library'] === false
```

Comment:
@knolleary not for this PR, but might be nice to build a small helper function on `this.team` called `sharedLibraryAvailable()` to hide the internals of the team.

Reply:
`this.team` is _just_ an object from API response at this stage, not a `Team` class with it's own functions, etc.